docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -10,7 +10,7 @@ title: StatefulSet Basics
 ---
 
 {% capture overview %}
-This tutorial provides an introduction to managing applications with
+This tutorial provides an introduction to manage applications with
```

Comment:
pls change this back

Reply:
to doing in the Form？

Reply:
yes. This should be interpreted as

an introduction to <managing applications>

instead of 

an introduction <to manage application>